code/index.html

```diff
@@ -1,30 +1,93 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>🐶 Dogpool</title>
+</head>
+<body>
+    <header> 
+      
+        <div class="header-video" style="width: 1349px; height: 800px;">
+             <video autoplay="true" loop="true" muted="" id="header-video"> type="video/mp4">
+            <source src="C:\Users\siri_\Desktop\KodÖvEx\Pexels Videos 2776522.mp4" type="video/mp4"></video></div>
```

Comment:
The source of the video isn't correct. So you need the path to be from this page you are on. So the video and `index.html` is in the same file so you only need `video.mp4` as your `src`. A tip is to rename your video file to something easier and without spaces!